src/cairo_interaction.rs

```diff
@@ -10,10 +10,10 @@ use ffi::{self, GdkRGBA};
 use super::{Pixbuf, Window};
 use cairo::{Context, RectangleInt};
 
-//pub fn create_region_from_surface() { }
+//pub fn create_region_from_surface()
```

Comment:
The included script doesn't attempt to modify the files it only prints any offending functions.